### ISSUE TITLE
SpellChecker throws exception in Visual Studio 17.12.1

### ISSUE THREAD

**Issue by JorisCleVR** [EWSoftware/VSSpellChecker#316]
**Reproduction steps:**
- Install Visual Studio 17.12.1
- Install Visual Studio Spell Checked 2024.6.17.0 as an extension

**In the Error List of visual studio the following warning is logged:**
Analyzer 'VisualStudio.SpellChecker.CodeAnalyzer.CSharpSpellCheckCodeAnalyzer' threw an exception of type 'System.IO.FileNotFoundException' with message 'Could not load file or assembly 'VisualStudio.SpellChecker.Common, Version=2021.7.27.0, Culture=neutral, PublicKeyToken=c626d3cbee5c69d2'. The system cannot find the file specified.'.
Exception occurred with following context:
Compilation: CleVR.WPF
SyntaxTree: C:\Git\Library\WPF\WPF\BaseCommandLineHandler.cs
System.IO.FileNotFoundException: Could not load file or assembly 'VisualStudio.SpellChecker.Common, Version=2021.7.27.0, Culture=neutral, PublicKeyToken=c626d3cbee5c69d2'. The system cannot find the file specified.
File name: 'VisualStudio.SpellChecker.Common, Version=2021.7.27.0, Culture=neutral, PublicKeyToken=c626d3cbee5c69d2'
   at VisualStudio.SpellChecker.CodeAnalyzer.CSharpSpellCheckCodeAnalyzer.AnalyzeSyntaxTree(SyntaxTreeAnalysisContext context)
   at Microsoft.CodeAnalysis.Diagnostics.AnalyzerExecutor.ExecuteAndCatchIfThrows_NoLock[TArg](DiagnosticAnalyzer analyzer, Action`1 analyze, TArg argument, Nullable`1 info, CancellationToken cancellationToken)

Suppress the following diagnostics to disable this analyzer: VSSpell001, VSSpell002

**Comment by EWSoftware** [comment]
I'm not seeing any errors in the error list but the code analyzer has definitely stopped working.  The tagger that handles comments and strings still works.  Trying to build the project fails with some odd error in the build tools SDK package.  It looks like Microsoft has made some breaking changes to the SDK again.

**Comment by rafal-zieba** [comment]
It is broken like this since 17.12.0. I was hoping 17.12.1 would fix it but no, it hasn't.

**Comment by scottkuhl** [comment]
Seeing the same issue here as well.

**Comment by LstpierreMaximum** [comment]
same for me.. 
by the way.. thanks for this good tool

**Comment by EWSoftware** [comment]
Something in the SDK changed and I had to mark the spell checker assemblies as not being CLS compliant.  There was also a change when running out of process where the host now makes a copy of the code analyzer assembly in a cache folder but doesn't include any of the dependency assemblies to it fails to find them.  I had to come up with a hack to work around that so it's now spell checking code elements again.  The new release will be available shortly.
